build/transifex/pom.xml

```diff
@@ -0,0 +1,640 @@
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/system-status.yaml</inputSpec>
+              <output>${project.build.directory}/api/system-status</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>templates</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/templates.yaml</inputSpec>
+              <output>${project.build.directory}/api/templates</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>transforms</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/transforms.yaml</inputSpec>
+              <output>${project.build.directory}/api/transforms</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>usergroup</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/usergroup.yaml</inputSpec>
+              <output>${project.build.directory}/api/usergroup</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>wmslayers</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/wmslayers.yaml</inputSpec>
+              <output>${project.build.directory}/api/wmslayers</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>wmsstores</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/wmsstores.yaml</inputSpec>
+              <output>${project.build.directory}/api/wmsstores</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>wmtslayers</id>
+            <phase>process-resources</phase>
+            <goals><goal>generate</goal></goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/wmtslayers.yaml</inputSpec>
+              <output>${project.build.directory}/api/wmtslayers</output>
+            </configuration>
+          </execution>
+          <execution>
+            <id>wmtsstores</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <inputSpec>${project.basedir}/api/1.0.0/wmtsstores.yaml</inputSpec>
```

Comment:
What's happening in this POM? 
Are you trying to provide translations for the REST API definitions as well?

Reply:
No I simply copy/paste the POM from the documentation projet and didn't replace it. I will remove it from now.

Reply:
I was checking if I could merge, but the pom.xml seems to be still there.

Reply:
Sorry Andrea. I just did it.